packages/wrapped-keys/src/lib/service-client/utils.ts

```diff
@@ -104,29 +105,40 @@ async function getResponseJson<T>(response: Response): Promise<T | string> {
   }
 }
 
+export function generateRequestId(): string {
+  return Math.random().toString(16).slice(2);
+}
+
 export async function makeRequest<T>({
   url,
   init,
+  requestId,
 }: {
   url: string;
   init: RequestInit;
+  requestId: string;
 }) {
-  console.log('Fetching from URL', url);
-  const response = await fetch(url, { ...init });
+  try {
+    const response = await fetch(url, { ...init });
 
-  if (!response.ok) {
-    const errorMessage = await getResponseErrorMessage(response);
-    throw new Error(`Failed to make request for wrapped key: ${errorMessage}`);
-  }
+    if (!response.ok) {
+      const errorMessage = await getResponseErrorMessage(response);
+      throw new Error(`HTTP(${response.status}): ${errorMessage}`);
```

Comment:
It's good to keep a common searchable text for all failed Wrapped Key operations?
```suggestion
      throw new Error(`HTTP(${response.status}): Failed to make request for wrapped key: ${errorMessage}`);
```

Reply:
This error is an 'inner' error that is caught below in the block L136-L142.  We will insert this string _into_ the message on the error we throw on L137, which prefixes this string with consistent wording: `Request(${requestId}) for wrapped key failed. Error:`